src/sumstyle.sh

```diff
@@ -2,6 +2,22 @@
 #
 # Summarize the formatting style metrics for the specified directory
 #
+if ! [ -x "$(command -v qmcalc)" ]; then
+  echo 'Error: qmcalc is not installed.' >&2
+  exit 1
+fi
+if [ "$1" = "-h" ] || [ $# -eq 0 ]; then
+  echo "Usage: `basename $0` <target directory> -- processes all C files in the directory
```

Comment:
Output description belongs to a manual page, not usage information. This is not a big deal though.

Reply:
done